nautobot/core/settings.py

```diff
@@ -189,6 +189,7 @@
         "rest_framework.renderers.JSONRenderer",
         "nautobot.core.api.renderers.FormlessBrowsableAPIRenderer",
     ),
+    "DEFAULT_PARSER_CLASSES": ("rest_framework.parsers.JSONParser",),
```

Comment:
```suggestion
    "DEFAULT_PARSER_CLASSES": (
        "rest_framework.parsers.JSONParser",
        "rest_framework.parsers.FormParser",
        "rest_framework.parsers.MultiPartParser",
    ),
```
This should fix the test failure [here] (https://github.com/nautobot/nautobot/actions/runs/3500386878/jobs/5863094878#step:7:151). Alternatively you can delete the entire `DEFAULT_PARSER_CLASSES`, and it should still pass the tests but I m not too sure what the impact would be to the rest of the code change.

Reply:
This was addressing this feedback: https://github.com/nautobot/nautobot/pull/2644#discussion_r1025697159

Reply:
I see you can do this then:
```
index d6d024e35..686c96516 100644
--- a/nautobot/core/views/mixins.py
+++ b/nautobot/core/views/mixins.py
@@ -22,6 +22,7 @@ from django.views.generic.edit import FormView
 from rest_framework import mixins, exceptions
 from rest_framework.response import Response
 from rest_framework.viewsets import GenericViewSet
+from rest_framework.parsers import JSONParser, MultiPartParser
 
 from drf_spectacular.utils import extend_schema
 
@@ -65,6 +66,7 @@ class NautobotViewSetMixin(GenericViewSet, AccessMixin, GetReturnURLMixin, FormV
     """
 
     renderer_classes = [NautobotHTMLRenderer]
+    parser_classes = [JSONParser, MultiPartParser]
     logger = logging.getLogger(__name__)
     lookup_field = "slug"
```

Reply:
https://github.com/nautobot/nautobot/pull/2644/commits/8e95cf6cdaaf774f49976a819ef954f17de365a9